platform/polkadot/api.go

```diff
@@ -0,0 +1,67 @@
+package polkadot
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+type Platform struct {
+	client    Client
+	CoinIndex uint
+}
+
+func (p *Platform) Init() error {
+	p.client = Client{blockatlas.InitClient(viper.GetString(p.ConfigKey()))}
+	return nil
+}
+
+func (p *Platform) Coin() coin.Coin {
+	return coin.Coins[p.CoinIndex]
+}
+
+func (p *Platform) ConfigKey() string {
+	return fmt.Sprintf("%s.api", p.Coin().Handle)
+}
+
+func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
+	s, err := p.client.GetTransfersOfAddress(address)
+	if err != nil {
+		return nil, err
+	}
+
+	txs := make([]blockatlas.Tx, 0)
+	for _, srcTx := range s {
+		tx := p.NormalizeTransfer(&srcTx)
+		txs = append(txs, tx)
+	}
+
+	return txs, nil
+}
+
+func (p *Platform) NormalizeTransfer(srcTx *Transfer) blockatlas.Tx {
```

Comment:
Can you call it NormalizeTransaction?

Reply:
they're called `Transfer` and `Extrinsic` and from two apis, `NormalizeExtrinsic` also added